templates/list-action.tmpl

```diff
@@ -6,7 +6,7 @@
       {% if content.block.cancel_text %}
         {{ content.block.cancel_text }} <br>
       {% endif %}
-      <a id="cancel-and-return" href="{{ previous_location_url }}">{{ _("Return to the previous page") }}</a>
+      <a id="cancel-and-return" href="{{ previous_location_url }}">{{ _("Cancel and return to the previous page") }}</a>
```

Comment:
The link has been moved in the new design. It now needs to appear between the last answer and the submit button:
![image](https://user-images.githubusercontent.com/1754822/80697060-4ee9d780-8ad0-11ea-9481-590639af1766.png)

Reply:
So  I was told it is an incorrect design, it stays where it is now.